recipes/bumpver/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "bumpver" %}
+{% set version = "2021.1112" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bumpver-{{ version }}.tar.gz
+  sha256: 715dd9e8526609de2008a9c62ec80d891af41055f95e6c328a8ca6dc8f0fa54a
+
+build:
+  number: 0
```

Comment:
Any reason not to make this `noarch: python`?

Reply:
nope there is no reason. added